lib/src/tag/tagger.dart

```diff
@@ -355,6 +350,53 @@ class Tagger {
     }
   }
 
+  void swiftPackageManagerPluginTag(
+      List<String> tags, List<Explanation> explanations) {
+    if (!_usesFlutter) return;
+    final pubspec = _pubspecCache.pubspecOfPackage(packageName);
+
+    bool pathExists(dynamic m, List<String> path) {
+      dynamic current = m;
+      for (final e in path) {
+        if (current is! Map) return false;
+        if (!current.containsKey(e)) return false;
+        current = current[e];
+      }
+      return true;
+    }
+
+    var isDarwinPlugin = false;
+    var swiftPmSupport = true;
+
+    for (final darwinOs in ['macos', 'ios']) {
+      if (pathExists(
+          pubspec.originalYaml, ['flutter', 'plugin', 'platforms', darwinOs])) {
+        isDarwinPlugin = true;
+        final specificPackageSwiftFile =
+            path.join(darwinOs, packageName, 'Package.swift');
+        final genericPackageSwiftFile =
+            path.join('darwin', packageName, 'Package.swift');
+        if (!(File(path.join(packageDir, specificPackageSwiftFile))
+                .existsSync() ||
+            File(path.join(packageDir, genericPackageSwiftFile))
+                .existsSync())) {
+          swiftPmSupport = false;
+          explanations.add(Explanation(
+              'Package does not support the Swift Package Manager on $darwinOs',
```

Comment:
Super nitpick: could we use the `iOS` and `macOS` casings here? I don't feel strongly